recipe/meta.yaml

```diff
@@ -1,47 +1,129 @@
-{% set name = "mesaLib" %}
-{% set version = "17.2.0" %}
+{% set version = "25.1.0" %}
 
 package:
-  name: {{ name|lower }}
-  version: "{{ version }}"
+  name: mesalib
+  version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: ftp://ftp.freedesktop.org/pub/mesa/mesa-{{ version }}.tar.gz
-  sha256: 9484ad96b4bb6cda5bbf1aef52dfa35183dc21aa6258a2991c245996c2fdaf85
+  url: https://archive.mesa3d.org/mesa-{{ version }}.tar.xz
+  sha256: b1c45888969ee5df997e2542654f735ab1b772924b442f3016d2293414c99c14
 
 build:
-  skip: True  # [win or py3k]
   number: 0
+  skip: true  # [win]
   track_features:
-      - mesalib
+    - mesalib
+  run_exports:
+    - {{ pin_subpackage('mesalib', max_pin='x.x') }}
 
 requirements:
   build:
-      - pthread-stubs
-      - pkgconfig
-      - gcc  # [linux]
-      - toolchain
-      - llvmdev
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('rust') }}
+    - meson >=0.63.0
+    - ninja
+    - pyyaml
+    - mako
+    - packaging
+    - python 3.12
+    - flex
+    - bison
+    - pkg-config
+    - llvm
+    - llvmdev  # [build_platform != target_platform]
+    - glslang
+    - git  # [linux]
+  host:
+    - expat
+    - zlib
+    - zstd
+    - libdrm  # [linux]
+    - spirv-tools 2025.1
+    - llvm 17.0.6
+    - llvmdev 17.0.6
+    - libxcb  # [linux]
+    - xorg-libx11  # [linux]
+    - xorg-xorgproto  # [linux]
+    - xorg-libxrandr  # [linux]
+    - xorg-libxshmfence  # [linux]
+    - xorg-libxfixes 6.0.1  # [linux]
+    - xorg-libxdamage  # [linux]
+    - xorg-libxxf86vm  # [linux]
+    - libgl-devel  # [linux]
+    - libglvnd  # [linux]
+    - libglvnd-devel  # [linux]
+    - libunwind 1.5.0  # [linux]
   run:
-      - pthread-stubs
-      - libgcc  # [linux]
-      - llvmdev
+    - expat
+    - zlib
+    - zstd
+    - libdrm  # [linux]
+    - spirv-tools
+    - llvm
+    - llvmdev
+    - libxcb  # [linux]
+    - xorg-libx11  # [linux]
+    - xorg-xorgproto  # [linux]
+    - xorg-libxrandr  # [linux]
+    - xorg-libxshmfence  # [linux]
+    - xorg-libxfixes  # [linux]
+    - xorg-libxdamage  # [linux]
+    - xorg-libxxf86vm  # [linux]
+    - libglvnd  # [linux]
```

Comment:
```suggestion
    - xorg-libx11  # [linux]
    - xorg-libxrandr  # [linux]
    - xorg-libxshmfence  # [linux]
```

Reply:
Fixed

Reply:
```
WARNING (mesalib): run-exports library package defaults/linux-64::xorg-libxrandr==1.5.4=h9b100fa_0 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (mesalib): run-exports library package defaults/linux-64::libunwind==1.5.0=h295c915_1 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
```

I removed `xorg-libxrandr` from `run` and added `libunwind` to `ignore_run_exports`.